### PR TITLE
implemented Material UI Typography and resized modal and image

### PR DESCRIPTION
**Pull request by PatrickMetsch**

### CODE REVIEW

File: src/components/forms/HPButton.tsx [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+  createStyles({
+    button: {
+      margin: theme.spacing(1),
+      backgroundColor: 'red',
+      color: 'white',
+      '&:hover': {
+        backgroundColor: '#ff5252'
+      }
+    },
+    input: {
```
Comment (bob-bins) [inline src/components/forms/HPButton.tsx]: what's this `input` section for? it's not explicitly referenced anywhere

File: src/components/site/SiteWithoutStyle.tsx [hunk 2]
```diff
@@ -9,10 +9,10 @@ export const SiteWithoutStyle = (props: SiteProps) => {
   const { shouldShowCelebration, onCelebrationXClicked, isSideNavOpen, url } = props;
   const { classes, ...propsWithoutClasses } = props;
   return <React.Fragment>
+  {1 < 2 && <CelebrationModal numHeartpointsAwarded={10} onXClicked={onCelebrationXClicked} />}
```
Comment (bob-bins) [inline src/components/site/SiteWithoutStyle.tsx]: what's `1 < 2 ` for?

Comment (PatrickMetsch) [reply src/components/site/SiteWithoutStyle.tsx]: Just a dumb way to keep the modal open without having to click through the Facebook login process every time I rebuilt.

I will change it back to `{shouldShowCelebration && ...` on my next commit. Sorry. I should have removed it before pushing.

File: src/components/forms/HPButton.tsx (hunk 1 above)
Comment (PatrickMetsch) [reply src/components/forms/HPButton.tsx]: Double checking now. That may have been a lack of due diligence on my part when copying over a solution from Stackoverflow.

**Pull request merged**